NitroxPatcher/Patches/Dynamic/FootstepSounds_OnStep_Patch.cs

```diff
@@ -61,4 +74,27 @@ private static float CalculateVolume(float originalVolume, FootstepSounds instan
         Resolve<FMODWhitelist>().TryGetSoundData(EXO_STEP_SOUND_PATH, out SoundData soundData);
         return soundData.Radius;
     });
+    public static void SendFootstepPacket(FMODAsset asset)
+    {
+        // Send footstep packet with the asset
```

Comment:
Comment not needed, explained by method name and parameter

Reply:
Done